web_poet/__init__.py

```diff
@@ -1,2 +1,14 @@
 from .pages import WebPage, ItemPage, ItemWebPage, Injectable
-from .page_inputs import HttpResponse, HttpResponseBody, HttpResponseHeaders
+from .requests import (
+    request_backend_var,
+    HttpRequest,
+    HttpRequestHeaders,
```

Comment:
I wonder if these objects can be page inputs as well. E.g. can a page object  for the request headers?

Reply:
Yeah, that makes sense. I imagine there would be some cases wherein developers need both the **request-response** pairs inside the Page Object.

We'll need to add a definition for this in `pages.py`. For now, however, I simply moved them into `page_inputs.py` in https://github.com/scrapinghub/web-poet/pull/22/commits/99e9b41cce3494c159d7f33df27897eb61fff38b. This simplifies the tests as well since they have a lot in common.